Remotion/Data/DomainObjects.RdbmsTools/RdbmsToolsRunner.cs

```diff
@@ -30,22 +29,24 @@
 using Remotion.Data.DomainObjects.Queries.Configuration;
 using Remotion.Logging;
 using Remotion.Tools;
+#if NETFRAMEWORK
+using System.IO;
+#endif
 
 namespace Remotion.Data.DomainObjects.RdbmsTools
 {
   /// <summary>
   /// The <see cref="RdbmsToolsRunner"/> type contains the encapsulates the execution of the various functionality provided by the 
   /// <b>Remotion.Data.DomainObjects.RdbmsTools</b> assembly.
   /// </summary>
+#if NETFRAMEWORK
   [Serializable]
   public class RdbmsToolsRunner : AppDomainRunnerBase
+#else
+  public class RdbmsToolsRunner : CustomAppContextRunnerBase
+#endif
   {
-    public static RdbmsToolsRunner Create (RdbmsToolsParameters rdbmsToolsParameters)
-    {
-      AppDomainSetup appDomainSetup = CreateAppDomainSetup(rdbmsToolsParameters);
-      return new RdbmsToolsRunner(appDomainSetup, rdbmsToolsParameters);
-    }
-
+#if NETFRAMEWORK
     public static AppDomainSetup CreateAppDomainSetup (RdbmsToolsParameters rdbmsToolsParameters)
```

Comment:
Okay, you changed to get rid of the factory method / composition. That's a breaking change we don't need. It probably won't hurt anyone, though, so I don't really care. 

Should we make the helper method internal. Marcel added an internals-visible-to automatic for unit tests, not sure if it's globally hooked in though.